tsl/test/sql/chunk_api.sql

```diff
@@ -70,6 +70,12 @@ FROM _timescaledb_functions.show_chunk(:'CHUNK_SCHEMA'||'.'||:'CHUNK_NAME') \gse
 
 TRUNCATE chunkapi;
 
+-- adding an existing table with additional columns should fail
+\set ON_ERROR_STOP 0
+CREATE TABLE extracol_chunk (time timestamptz NOT NULL, device int, temp float, extra_col int);
+SELECT * FROM _timescaledb_functions.create_chunk('chunkapi', :'SLICES', :'CHUNK_SCHEMA', :'CHUNK_NAME', 'extracol_chunk');
```

Comment:
I am not sure what happens if you have correctly named chunks, but in different order than the hypertable.